TripHouse/main.js

```diff
@@ -51,17 +51,60 @@ const data = [
     },
   ];
 
-const homesContent = document.querySelector('.homes_content')
-
+const homesContent = document.querySelector('.homes_content');
 function createCard1(data) {
   return `
   <div class="content_img col-xs-6 col-sm-6">
     <img src="${data.imageUrl}" alt="${data.name}">
     <p class="accent-blue">${data.name}</p>
     <p class="secondary">${data.city}, ${data.country}</p>
   </div>`
+};
+data.forEach((element, index) => {
+  if (index < 4) {
+    homesContent.insertAdjacentHTML('beforeend', createCard1(element))
+  }
+});
+
+const homesImgArrowRight = (homesContent.getElementsByClassName('content_img')[3]);
+function createArrowRigt() {
+  return `<img class="arrow_right" src="./img/arrow-right.png" alt="arrow-right">`
+};
+homesImgArrowRight.insertAdjacentHTML('beforeend', createArrowRigt())
+
+const homesImgArrowLeft = (homesContent.getElementsByClassName('content_img')[0])
+function createArrowLeft() {
+  return `<img class="arrow_left" src="./img/arrow-left.png" alt="arrow-leftt">`
+};
+homesImgArrowLeft.insertAdjacentHTML('beforeend', createArrowLeft());
+
+const slider = document.querySelector('.homes_content');
+const slides = document.querySelectorAll('.content_img')
+const next = document.querySelector('.arrow_right');
+const prev = document.querySelector('.arrow_left');
+
+let position = 0;
+
+if (position === 0) {
+	prev.style.display = 'none';
 }
 
-data.forEach(element => {
-  homesContent.insertAdjacentHTML('beforeend', createCard1(element))
-})
\ No newline at end of file
+// next.addEventListener('click', function() {
+// 	position	-= slides[0].offsetWidth;
+// 	if (position  < 0) prev.style.display = 'block'
+// 	if (position === -slider.offsetWidth + slides[0].offsetWidth) {
+// 		next.style.display = 'none';
+// 		position = -slider.offsetWidth + slides[0].offsetWidth;
+// 	}
+// 	slider.style.transform = `translateX(${position}px)`;
+// })
+
+// prev.addEventListener('click', function() {
+// 	position	+= slides[0].offsetWidth;
+// 	if (position >= -slider.offsetWidth + slides[0].offsetWidth) next.style.display = 'block';
+// 	if (position === 0) {
+// 		prev.style.display = 'none';
+// 		position = 0;
+// 	}
+// 	slider.style.transform = `translateX(${position}px)`;
+// })
```

Comment:
Why did you commented listeners? Should I wait for you will finish it ? Or should I marge it now?